src/bidiMapper/modules/network/CollectorsStorage.ts

```diff
@@ -84,7 +94,12 @@ export class CollectorsStorage {
     return true;
   }
 
-  disown(requestId: Network.Request, collectorId?: string) {
+  isRequestCollected() {
```

Comment:
If you want to have this let's do the correct types so 

```suggestion
  isRequestCollected(_requestId: Network.Request, _collectorId?: string) {
```